### DISCUSSION TITLE
[FR] Pop-up height based on content

### DISCUSSION THREAD

**Discussion by CZonin** [Clooos/Bubble-Card#841] [Feature requests]
I've manually set the pop-up top margin for mobile and desktop for all of my pop-ups to match the content inside them. I have some where the content inside is conditional so certain items show/hide based on specific criteria. With those, it's not possible for me to match the pop-up height since things constantly change.

I was wondering if it would be possible to have the height of the pop ups match the content inside of them? I feel like that would be a much simpler way for users to set the height of the pop-ups, and should potentially be the default setting.

**Comment by RobertoCravallo** [comment]
+1

**Reply by ronluna** [reply]
+1

**Reply by ericstammen** [reply]
+1

**Comment by Clooos** [comment]
Hi! Just wanted to say that this feature is (finally) coming in the upcoming v3.2.0 😄

**Reply by CZonin** [reply]
Looking forward to it!

**Reply by CZonin** [reply]
Just started testing it with 3.2.0-beta.3 and it's working great so far. Thanks for adding it @Clooos !

**Reply by Clooos** [reply]
Thank you so much for the positive feedback! ❤️
